### DISCUSSION TITLE
When Will "Version 12" add to Maven Central ?

### DISCUSSION THREAD

**Discussion by krdk01** [graphql-java-kickstart/graphql-spring-boot#708] [Q&A]
I want to use graphql  v.12 as you point out  in your documentation.
https://mvnrepository.com/artifact/com.graphql-java-kickstart/graphql-spring-boot-starter

I'm getting this error :
 "Could not find artifact com.graphql-java-kickstart:graphql-spring-boot-starter:pom:12.0.0 in central" 

When I try wiht  snapshoot also I'm getting this error: 
 "Could not find artifact com.graphql-java-kickstart:graphql-spring-boot-starter:pom:12.0.0 in osshr-snapshots"

How can I use version 12 from Maven Central?
Thank you.

**Comment by yilmazarmagan** [comment]
I'm getting the same error too. is there anyone who can help?

**Reply by setchy** [reply]
All things going well, this weekend - https://github.com/graphql-java-kickstart/graphql-spring-boot/issues/676#issuecomment-919747919
